pkg/status/templates/logsagent.tmpl

```diff
@@ -19,7 +19,7 @@ Logs Agent
 
 {{- if and (eq .use_http false) (eq .is_running true) }}
 
-    You are currently sending Logs to Datadog through TCP. To benefit from increased reliability and better network performances, we strongly encourage to switch over compressed HTTPS by using the logs_config.use_http and logs_config.use_compression parameters. This will become the default protocol in the Agent future version.
+    You are currently sending Logs to Datadog through TCP either because logs_config.use_tcp is set or HTTP connectivity test has failed. To benefit from increased reliability and better network performances, we strongly encourage to switch over compressed HTTPS which is now the default protocol.
```

Comment:
```suggestion
    You are currently sending Logs to Datadog through TCP (either because logs_config.use_tcp is set or the HTTP connectivity test has failed). To benefit from increased reliability and better network performances, we strongly encourage switching over to compressed HTTPS which is now the default protocol.
```